server/src/routes/routes.js

```diff
@@ -1,17 +1,21 @@
 import express from 'express';
 import signupValidator from '../validators/signupValidator';
 import loginValidator from '../validators/loginValidator';
-import userFieldsErrors from '../middleware/userFieldsErrors';
+import fieldsErrors from '../middleware/fieldsErrors';
 import userController from '../controllers/userController';
 import auth from '../middleware/auth';
+import leaderboardController from '../controllers/leaderboardController';
+import leaderboardValidator from '../validators/leaderboardValidator';
 
 const router = express.Router();
 
 router.get('/ping', (req, res) => {
   res.json({ status: 'working' });
 });
-router.post('/api/signup', signupValidator, userFieldsErrors, userController.createNewUser);
-router.post('/api/login', loginValidator, userFieldsErrors, userController.loginUser);
+router.post('/api/signup', signupValidator, fieldsErrors, userController.createNewUser);
+router.post('/api/login', loginValidator, fieldsErrors, userController.loginUser);
 router.get('/api/validate', auth, (req, res) => res.send(req.user));
+router.post('/api/leaderboard', auth, leaderboardValidator, fieldsErrors, leaderboardController.addScore);
```

Comment:
the `auth` variable should be called `requireAuthentication` or something so its more clear what it is doing.

Reply:
Yeah, okay.